containers/shop/product-card.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+  <div class="card product with-image">
+    <Button
+      v-if="closable"
+      @click="closeModal"
+      normal
+      round
+      class="close"
+      img="/images/x.svg"
+    />
+    <Carousel :images="activeVariety.images" />
+    <div class="content">
+      <div class="title">{{ name }}</div>
+      <div class="subtitle">{{ type }}</div>
+
+      <Labeled class="text card-row" label="Description">
+        {{ description }}
+      </Labeled>
+
+      <div class="card-row">
+        <Labeled class="text" label="Colors">
+          <RadioGroup
+            :items="varieties"
+            :name="'color-' + id"
+            v-model="activeVariety"
+            class="radio-options"
+          />
+        </Labeled>
+
+        <div class="labeled text">
+          <span class="label">Sizes</span>
+          <div role="group" class="radio-options">
+            <Chip
+              v-for="(val, key) in activeVariety.sizes"
+              :disabled="val === 0"
+              :value="key === chosenSize"
+              :key="key"
+              @click="chooseSize(key)"
```

Comment:
Event name is "input". That's why the chosen size doesn't get updated
```suggestion
              @input="chooseSize(key)"
```
After that there is the edge case of choosing a size and then switching to a color where this size isn't available; the `chosenSize` remains the same. Need to reset `chosenSize` when choosing a different color

Reply:
Done